framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -957,6 +960,73 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
     }
 
     /**
+     * A helper method to check that the array type of actual varargs is a subtype of the
+     * corresponding required varargs, and issues "argument.invalid" error if it's not a subtype of
+     * the required one.
+     *
+     * <p>Note it's required that type checking for the each elements of varargs is executed by
```

Comment:
`for the each elements of varargs` -> `for each element in varargs`
`by caller` -> `by the caller`